python/cudf/cudf/utils/utils.py

```diff
@@ -222,6 +222,11 @@ def set_allocator(
     )
 
 
+def clear_cache():
```

Comment:
Why are we adding this public API? Was this just for testing / can it be removed now?

Reply:
Ah, I apologize. I did a Ctrl+F for "clear_cache" but I had already marked the `__init__.py` where this is used (in the RMM reinitialize hook) as "Viewed." This can remain as-is, but it _could_ be a private API since we only use it internally.